MoleculeVAE-reImplementation/train.py

```diff
@@ -279,57 +285,102 @@ def plot3D(xi, yi, zi, c, label, title, n_min, n_max):
 
 '''Train & Test func.'''
 
-def train(epochs,train_set_final, valid_set_final,logger):
+def train(
+        model: torch.nn.Module,
```

Comment:
Changed to passing in `model` as an input. As with `device` this makes the dependency explicit, which I think is clearer.